pipeline.py

```diff
@@ -15,6 +15,8 @@
 from src.production_file import ProductionFile
 from src.translate_rapid_pro_keys import TranslateRapidProKeys
 
+log = Logger(__name__)
```

Comment:
I think we're also going to need the project name in the logs. They're going to end up being sent to a logs aggregator that won't know which project `__main__` is in. Consider either ADSS/`__name__` or full path (which might get a bit on the verbose side and make the all the logs spill onto multiple lines)

Reply:
Made an issue for this https://github.com/AfricasVoices/CoreDataModules/issues/128